warehouse/static/js/warehouse/utils/doc-ready.js

```diff
@@ -11,7 +11,29 @@
  * limitations under the License.
  */
 
+import unsupportedBrowser from "./unsupported_browser";
+
+let warningShown = false;
+
+function showUnsupportedBrowserWarning() {
+  if (document.getElementById("unsupported-browser") !== null) return;
+
+  let warning_html = "<div id='unsupported-browser' class='notification-bar notification-bar--danger'><span class='notification-bar__icon'><i class='fa fa-exclamation-triangle' aria-hidden='true'></i><span class='sr-only'>Warning:</span></span><span class='notification-bar__message'>You are using an unsupported browser, please upgrade to a newer version.</span></div>";
+  let warning_div = document.createElement("div");
+  warning_div.innerHTML = warning_html;
+
+  let warning_section = document.getElementById("stick-to-top");
+  warning_section.appendChild(warning_div);
+  warningShown = true;
+}
+
 export default (fn) => {
+  if (unsupportedBrowser) {
```

Comment:
Is this correct? `unsupportedBrowser` is a function, shouldn't we be calling it here?